components/index/system76.vue

```diff
@@ -21,7 +21,7 @@
         rel="noopener"
         title="Lemur"
       >
-        <sys-product-thumbnail product="lemp9" />
+        <img src="~/assets/images/index/thumbnail_lemp9.png">
```

Comment:
Do we want to move away from using these components? Don't they do some of the magic for us?

Reply:
They do some resizing magic, but we don't want to use the actual thumnails. Marketing has custom images for these.